namespaces/live.cloud-platform.service.justice.gov.uk/hmpps-incentives-prod/resources/irsa.tf

```diff
@@ -0,0 +1,52 @@
+
+module "irsa-api" {
+  source = "github.com/ministryofjustice/cloud-platform-terraform-irsa?ref=1.0.3"
+
+  namespace        = var.namespace
+  role_policy_arns = [aws_iam_policy.analytical-platform.arn]
+  service_account  = "${var.namespace}-api"
+  # NB: service account name must be unique within Cloud Platform (IAM role name is derived from it)
+}
+
+resource "aws_iam_policy" "analytical-platform" {
+  name   = "${var.namespace}-analytical-platform"
+  policy = data.aws_iam_policy_document.analytical-platform.json
+  # NB: IAM policy name must be unique within Cloud Platform
+
+  tags = {
+    business-unit          = var.business_unit
+    team_name              = var.team_name
+    application            = var.application
+    is-production          = var.is_production
+    namespace              = var.namespace
+    environment-name       = var.environment
+    owner                  = var.team_name
+    infrastructure-support = var.infrastructure_support
+  }
+}
+
+data "aws_iam_policy_document" "analytical-platform" {
+  # Allows direct put access to "landing" S3 bucket for Prison Network App in Analytical Platform AWS account (mojap)
```

Comment:
This policy is not actually for putting files into AP for Prison Network App use